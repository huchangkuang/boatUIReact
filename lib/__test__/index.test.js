import 'jsdom-global/register';
import renderer from "react-test-renderer";
import Adapter from 'enzyme-adapter-react-16';
import {configure, shallow} from "enzyme";
import {Button, Icon} from "../index";

configure({adapter: new Adapter()});
describe('icon', () => {
  it('should be svg', function () {
    const component = renderer.create(<Icon name='share' />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('should be div', function () {
    const component = renderer.create(<Button />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
})