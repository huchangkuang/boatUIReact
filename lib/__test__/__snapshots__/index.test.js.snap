// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icon should be div 1`] = `
<div
  className="boat-button"
>
  Button
</div>
`;

exports[`icon should be svg 1`] = `
<svg
  className="boat-icon"
>
  <use
    xlinkHref="#share"
  />
</svg>
`;
